defaults/main.yml

```diff
@@ -30,5 +30,5 @@ postgres_user_home: /var/lib/postgresql
 configure_docker: yes
 install_apparmor: yes
 install_rabbitmq: no
-docker_package: docker-engine
+docker_package: docker-ce
```

Comment:
just for info as it becomes the default, what is the difference between docker-ce and docker-engine ?

Reply:
My understanding is that `docker-engine` is the _old_ name that's been replaced with the `docker-ce` variant for the community edition: https://stackoverflow.com/a/45023650/523109
In turn, the `-engine` is not available on more recent operating system versions and vice versa.

Reply:
Good to know ! thanks